### ISSUE TITLE
Improve documentation on where the cloned repos are located

### ISSUE THREAD

**Issue by hello-stark** [maks/MGit#626]
**Is your feature request related to a problem? Please describe.**
The local files when downloaded are hidden from the files in android, especially when the search functionality of android is so bad, the repo i cloned were for the usage in another app, so viewing files inside the MGit app is pretty much useless. 

**Describe the solution you'd like**
I really would like t see a "view in file system" option or 
just a plain "show file location" inside the app so that we can open the files in the necessary software

**Describe alternatives you've considered**
One other possibility (as a developer which i know is neither easy nor that useful) is to allow direct integration like 
"open with
-gallery
-google photos
etc"

ie basically allow direct openings in proper apps. but this will be annoying if you are, as in my case, download markdown files to be used in obsidian app in multiple devices

**Additional context**
Context:
Basically, I use obsidian a lot for note taking, and I have a private repository where i commit the files which can be used between multiple devices, but sso fat, i was using 2 computers, so i would use normal git cli, recently i started using obsidian on phone, and needed a proper app for committing and pulling files across devices via git, and MGit is superb for the job, but one issue i am facing is, there is no way (in what i researched and experimented) to find the location where the cloned repo is stored so that can be opened in the obsidian app. The `local path` option while cloning the repo didnt seem to do anything.

Thanks in advance

**Comment by maks** [comment]
Thank you for opening this issue.
You can set the folder that is used by MGit for all local repos it manages by changing the `Root storage location for repos` in MGits `Settings` screen, eg. to `/storage/emulated/0/Documents`.

Installing Obsidian on Android 12, I see it requests the "All File Permission" so it should then be able to access any files in a repo you place in the Documents folder for example. However a problem you may run into if you use Android 11 or newer, is that MGit has not yet been granted use of that same permission by Google Play so it may not be able to access edited files itself, see #620 for more details on that issue.

**Comment by deephack1982** [comment]
Just tried exactly this and it works perfectly! Thanks for this great app!
